plugins/uefi/fu-uefi-common.c

```diff
@@ -272,6 +272,27 @@ fu_uefi_read_file_as_uint64 (const gchar *path, const gchar *attr_name)
 	return g_ascii_strtoull (data, NULL, 10);
 }
 
+gboolean
+fu_uefi_check_esp_path (const gchar *path, GError **error)
+{
+	g_autoptr(GUnixMountEntry) mount = g_unix_mount_at (path, NULL);
```

Comment:
Is this approach still going to work in a flatpak?  I'm not sure if bind mounts show up as mounts in the container or not.

Reply:
(yes i'm aware it was like that before, this is just a thought that came to my mind while looking at this PR)

Reply:
I believe so; it's been a few weeks since I tested it.

Reply:
Well the part that has me wondering is that the flatpak is built with the bindmount of `/boot`.  If the ESP is at `/boot/efi` the mount shows up at the wrong place, so this logic may fail.